typed-function.js

```diff
@@ -1428,6 +1449,33 @@
       typed.conversions.push(conversion);
     };
 
+    /**
+     * Check if an entity is a typed function created by this instance
+     * @param {any} entity
+     * @returns {boolean}
+     */
+    typed.isTypedFunction = entity => entity && typeof entity === 'function' &&
+      entity.typedFunctionToken === typed.uniqueToken
+
+    /**
+     * Return the specific signature that will be called by the given typed
+     * function on the given arguments. Here, a "signature" is an object with
+     * properties 'params', 'test', 'fn', and 'implementation' -- the latter
+     * converts params as necessary and then calls 'fn'. Returns null if
+     * the first argument is not a typed function, or if there is no matching
+     * signature.
+     * @param {TypedFunction} f
+     * @param {any[]} argList
+     * @returns {{params: string, test: function, fn: function, workingFn: function}}
```

Comment:
I think you renamed `workingFn` to `implementation` in the end.

I hadn't thought/realized it was possible to expose the `implementation` of the functions that take care of type conversions. That is really powerful!

Reply:
👍